exercises/parallel-letter-frequency/benches/benchmark.rs

```diff
@@ -0,0 +1,73 @@
+#![feature(test)]
+extern crate parallel_letter_frequency;
+extern crate test;
+
+use std::collections::HashMap;
+use test::Bencher;
+
+#[bench]
+fn bench_sequential(b: &mut Bencher) {
+    let texts = all_texts();
+    b.iter(|| frequency(&texts));
+}
+
+#[bench]
+fn bench_parallel(b: &mut Bencher) {
+    let texts = all_texts();
+    b.iter(|| parallel_letter_frequency::frequency(&texts, 3));
+}
+
+/// Simple sequential char frequency. Can it be beat?
+pub fn frequency(texts: &[&str]) -> HashMap<char, usize> {
```

Comment:
I see your point and I tend to agree, let's see what other members say. We are setting up a pattern here that we will use if we add benchmarks to other exercises.